### DISCUSSION TITLE
cemu not working with wiinusoft

### DISCUSSION THREAD

**Discussion by kamikazilucas** [KeyPuncher/WiinUPro#168] [Help]
controller works normally but when i start cemu it disconnects and whenever i try to connect it again it says failed to pair

**Comment by KeyPuncher** [comment]
If it's disconnecting then it's likely that cemu is trying to connect with it. I would suggest seeking help within the cemu community.

**Comment by hdfxgrd** [comment]
try with cemu stable version (1.26.2f)
